src/domain/events/eventActionsDropdown/EventActionsDropdown.tsx

```diff
@@ -127,7 +129,16 @@ const EventActionsDropdown: React.FC<EventActionsDropdownProps> = ({
           isOpen={openModal === EVENT_MODALS.CANCEL}
           isSaving={saving === EVENT_ACTIONS.CANCEL}
           onClose={closeModal}
-          onConfirm={cancelEvent}
+          onConfirm={() =>
+            cancelEvent({
+              onSuccess: () => {
```

Comment:
Is there some good reason that I don't know about for not also having an `onError` callback for displaying an error notification?